chapter1.md

```diff
@@ -640,6 +642,7 @@ Similar functions such as [`int()`](https://docs.python.org/3/library/functions.
 - Convert the variable `pi_string` to a float and store this float as a new variable, `pi_float`.
 
 `@hint`
+- The error message when you _Run Code_ for the first time is telling you that you have an error in line 6 where at least one of your variables is not a string.
 - You should use [`str()`](https://docs.python.org/3/library/functions.html#func-str) twice!
```

Comment:
@jasminludolf Similar to the previous comments, the links should be changed so they are not associated with code blocks.